CHANGELOG.md

```diff
@@ -18,8 +18,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - Serialization of `SendPoll::type_` (it's now possible to send quiz polls) ([#181][pr181])
+- `Update::chat()` now returns `Some(&Chat)` for `UpdateKind::ChatMember`, `UpdateKind::MyChatMember`,
```

Comment:
0.4.2 version is already released, this should go in "Unreleased" section

Reply:
fix